ql/src/test/org/apache/hadoop/hive/ql/udf/generic/TestGenericUDTFGetSQLSchema.java

```diff
@@ -84,6 +96,8 @@ public void testWithSimpleTypes() throws Exception {
 
   @Test
   public void testWithDDL() throws Exception {
+    // Set the execution engine to mr to avoid the NPE exception in stats flow
+    conf.set("hive.execution.engine", "mr");
```

Comment:
was this test not running earlier?

Reply:
@abstractdog do you have any pointers here? Can we live with it for now?

Reply:
in general, if a test fails with mr, and it's not convenient and easy to fix, it can stay on mr, but a ticket under HIVE-28027 is needed + comment like:
```
//TODO: HIVE-27998: TestGenericUDTFGetSQLSchema to run on Tez
```